01-getting-started/installation/platforms/install-on-azure-arm.md

```diff
@@ -0,0 +1,155 @@
+---
+
+# Deploy to Azure on Container Instances using ARM template
+
+This version of the installation tutorial deploys Squidex along with MongoDB (for datastore) and Caddy (for reverse proxy along with SSL) as a [Container Group](https://learn.microsoft.com/en-us/azure/container-instances/container-instances-container-groups) in Azure using an [Azure Resource Manager or ARM](https://learn.microsoft.com/en-us/azure/azure-resource-manager/templates/overview) template. The approach is called Infrastructure as Code.
+
+The tutorial does not cover the basics of Azure. You should be familiar with them before you begin with the installation instructions. 
+
+This tutorial runs MongoDB as a single container and is recommended for non-production or trial environments. For production environments it is recommended to use one of the MongoDB Atlas (or Enterprise) offerings from [Azure marketplace](https://azuremarketplace.microsoft.com/en-us/marketplace/apps?search=mongodb&page=1), which provides capabilities such as High Availability.
+
+Use this tutorial as a getting started guide in Azure to explore Squidex and use it as a Dev/Sandbox environment. This setup may not be recommended for a production environment. 
+
+## Pre-requisites
+
+Before you begin, ensure you meet the following requirements:
+
+1. An active Azure subscription.
+2. The following providers registered in your Azure subscription:
+   * Microsoft.ContainerInstance
+   * Microsoft.Storage 
+
+## Deployment Specifications & Other Details
+
+The ARM template creates the following Azure resources:
+* An Azure Storage Account
+  * Azure Container
+  * Azure Share
+* Azure Container Instances (Group)
+  * Squidex Web App Container (latest image)
+  * MongoDB Container (latest image)
+  * Caddy container (latest image)
+
+**Important Details**
+  * Default resource allocations
+    * Squidex container: **1 vCPU and 1.5 GiB memory**
+    * MongoDB container: **2 vCPU and 2 GiB memory**
+    * Caddy container: **1 vCPU and 1.5 GiB memory**
+  * Squidex and MongoDB containers remain private and are not exposed to public.
+  * Squidex talks to MongoDB without authentication (mongodb://localhost:27017). Since MongoDB is private it cannot be accessed from outside, hence secure to an extent. **With the current setup you cannot connect using an external client like MongoDB Compass. You can however connect from Azure Container Instance, click [here](#connecting-to-mongodb-instance) for instructions**
+  * Caddy reverse proxies to the Squidex Web App container.
+  * An FQDN is provided by Azure Container Instance, _Caddy_ provides SSL for this as the FQDN does not come with SSL. This FQDN is set as the `URLS__BASEURL` and `VIRTUAL_HOST` for Squidex configuration.
+
+**NOTE: If you wish to modify any of the variables such as vCPU, Memory etc., you can do so by editing the ARM template. Click [here](#modifying-arm-variables) for basic instructions.**
```

Comment:
For something like the notes here, you can use hints

```
{% hint style="info" %} YOUR HINT  {% endhint %
```

See: https://docs.squidex.io/01-getting-started/contributing-and-developing/developing

Reply:
Done